dist/etc/conf/nucleo-wl55jc.env

```diff
@@ -0,0 +1,84 @@
+HIL_DUT_RST_GPIO?=
+HIL_DUT_RST_PORT?=
+HIL_DUT_RST_PIN?=
+
+HIL_DUT_CTS_GPIO?=
+HIL_DUT_CTS_PORT?=
+HIL_DUT_CTS_PIN?=
+
+HIL_DUT_RTS_GPIO?=
+HIL_DUT_RTS_PORT?=
+HIL_DUT_RTS_PIN?=
+
+HIL_DUT_RX_GPIO?="PA2/CN9-7/D1"
```

Comment:
Are you sure this uart is not being used?

The fact that I get a timeout makes me suspect it is not just a wiring issue.

Reply:
> Below is the test report from the gpio test to ensure wiring, other than the
> SCL which was my fault I think there are some things to double check.
> 
> I would also like the labels of CN10 added as the connections to these are much cleaner to wire (I don't require an additional gender changing pin).

Okay. I have added references to Morpho connectors in latest commit.